clang/docs/StandardCPlusPlusModules.rst

```diff
@@ -691,7 +691,7 @@ ensure it is reachable, e.g. ``using N::g;``.
 Support for Reduced BMIs is still experimental, but it may become the default
 in the future. The expected roadmap for Reduced BMIs as of Clang 19.x is:
 
-1. ``-fexperimental-modules-reduced-bmi`` is opt-in for 1~2 releases. The period depends
+1. ``-fmodules-reduced-bmi`` is opt-in for 1~2 releases. The period depends
    on user feedback and may be extended.
 2. Announce that Reduced BMIs are no longer experimental and introduce
    ``-fmodules-reduced-bmi`` as a new option, and recommend use of the new
```

Comment:
w.r.t. keeping the "as of 19.x": since removing `experimental-` prefix was part of the plan since v19, the first point should either be marked as done (or simply removed), but not simply renamed (which would be inconsistent both with the "as of 19.x", as well as the following item that already covers the same point). For example:
```suggestion
1. ``-fexperimental-modules-reduced-bmi`` was introduced in v19.x
2. For v20.x, ``-fmodules-reduced-bmi`` is introduced as an equivalent non-experimental
   option. It is expected to stay opt-in for 1~2 releases, though the period depends
   on user feedback and may be extended.
```